Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -50,6 +50,10 @@
 #endif
 #import <QuartzCore/QuartzCore.h>
 
+#if MAC_OS_X_VERSION_MIN_REQUIRED < 1090
```

Comment:
```suggestion
#if defined(HS_BUILD_FOR_MACOS) && MAC_OS_X_VERSION_MIN_REQUIRED < 1090
```

Reply:
I think Apple defined `TARGET_OS_MAC` as well... feels a bit weird to mix and match Plasma and Apple macros, but that should work